src/tasks/minions/dungeoneeringActivity.ts

```diff
@@ -44,17 +44,20 @@ export default class extends Task {
 				bonusXP += Math.floor(xp * (gorajanEquipped / 2));
 				xp += bonusXP;
 			}
-			await u.addXP({
+			const xpStr = await u.addXP({
 				skillName: SkillsEnum.Dungeoneering,
 				amount: xp / 5,
-				duration
+				duration,
+				minimal: true
 			});
 			await u.settings.update(
 				UserSettings.DungeoneeringTokens,
 				u.settings.get(UserSettings.DungeoneeringTokens) + tokens
 			);
-			let rawXPHr = (xp / (duration / Time.Minute)) * 60;
-			rawXPHr = Math.floor(xp / 1000) * 1000;
+			let rawXPHr = (xp / minutes) * 60;
+			rawXPHr = Math.floor(rawXPHr / 1000) * 1000;
```

Comment:
Is this going to result in the xp/hr being shown twice in the message? I tried looking for your testing of this PR in the testing server, but couldnt find any dung related stuff.